modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotMessageCounter.java

```diff
@@ -56,6 +56,16 @@
 
     private static final int SLOT_SUBMIT_LOOP_SKIP_COUNT_THRESHOLD = 10;
 
+    /**
+     * Time between successive slot submit scheduled tasks.
+     */
+    public static final int SLOT_SUBMIT_TIMEOUT = 4000;
```

Comment:
Do we need to configure these externally? Anyway, we didn't have this configurable up to now.